pkg/karmadactl/token/token_test.go

```diff
@@ -0,0 +1,208 @@
+package token
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	kubernetesfake "k8s.io/client-go/kubernetes/fake"
+	"k8s.io/cluster-bootstrap/token/api"
+
+	"github.com/karmada-io/karmada/pkg/karmadactl/options"
+	karmadactltesting "github.com/karmada-io/karmada/pkg/karmadactl/testing"
+	"github.com/karmada-io/karmada/pkg/karmadactl/util"
+)
+
+type mockIOWriter struct {
+}
+
+func (m *mockIOWriter) Write(p []byte) (n int, err error) {
+	str := string(p)
+	fmt.Println(str)
+	return len(str), nil
+}
+
+func NewTestCommandTokenOptions() *CommandTokenOptions {
+	configFilePath := "tmp/kubeconfig.yaml"
+	context := "context-c1"
+	options.DefaultConfigFlags.KubeConfig = &configFilePath
+	options.DefaultConfigFlags.Context = &context
+	return &CommandTokenOptions{
+		TTL: &metav1.Duration{
+			Duration: time.Second,
+		},
+		Description:          "foo",
+		Groups:               []string{},
+		Usages:               []string{},
+		PrintRegisterCommand: true,
```

Comment:
I think we don't need to test the situation of `PrintRegisterCommand: true` here, it can be set in `pkg/karmadactl/util/bootstraptoken/bootstraptoken_test.go`.

We can refer to `kubeadm` at https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/cmd/token_test.go